docs/install_macosx.md

```diff
@@ -0,0 +1,49 @@
+#Mac OS X Installation Guide
+
+In OSX environment the intercept-build tool from
+[scan-build](https://github.com/rizsotto/scan-build) is used to log the
+compiler invocations.
+
+It is possible that the [intercept-build can not
+log](https://github.com/rizsotto/scan-build#limitations)
+the compiler calls without turning off *System Integrity Protection (SIP)*.
+`intercept build` can automatically detect if SIP is turned off.
+
+You can turn off SIP on El Capitan this way:
+
+  * Click the  (Apple) menu.
+  * Select Restart...
+  * Hold down command-R to boot into the Recovery System.
+  * Click the Utilities menu and select Terminal.
+  * Type csrutil disable and press return.
+  * Close the Terminal app.
+  * Click the  (Apple) menu and select Restart....
+
+The following commands are used to bootstrap CodeChecker on
+OS X El Capitan 10.11, macOS Sierra 10.12 and macOS High Sierra 10.13.
+
+```sh
+# Download and install dependencies.
+brew update
+brew install doxygen gcc git
+pip3 install virtualenv
+
+# Install the latest clang see: https://formulae.brew.sh/formula/llvm
+brew install llvm@7
```

Comment:
I think we can update the version here the current version is
```suggestion
brew install llvm@10.0.0
```
or to `brew install llvm` which should get always the latest.